### ISSUE TITLE
[feature request / bug] option to choose sim card for a call from the contact card and not only from the call history tab

### ISSUE THREAD

**Issue by avatar1024** [SimpleMobileTools/Simple-Dialer#693]
I have a dual sim. If I want to call someone that's not already in the call history, I have to go to the contact tab and the choose that contact. The problem is that from there there seem to be no way to choose the sim to call that contact. So I have to first initiate call (which initiates from the default sim card), then stop the call immediately, go to the call history tab where that contact now appears, click on the three little dot on the right hand side and then choose the other sim card. 

This feels like a highly convoluted workflow just to call someone :) so while technically a missing feature it also feels like a bug by omission ;).

Sorry if I have missed something obvious and it is in fact possible to do it another way!

**Comment by muxina-credo** [comment]
This is exactly the same issue that I've reported on issue #532 . A better workaround is mentioned in the later comments. 

You need to slide the notification menu and press the call you are trying to make from the contacts (careful not to hit the off button ); then the SIM prompt will appears; then in my case you have to choose your SIM 3 times before it works.

I really looks like a bug but I don't know.

**Comment by avatar1024** [comment]
I don't think that's quite the same problem. I when through your bug report and I cannot reproduce. For me initiating calls works just fine from any places. It's just that I cannot choose the SIM, unless I initiate from the Call History tab.

As for the workaround you mentioned, I don't understand what you mean by "You need to slide the notification menu and press the call you are trying to make from the contacts (careful not to hit the off button)".

**Comment by muxina-credo** [comment]
> I don't think that's quite the same problem. I when through your bug report and I cannot reproduce. For me initiating calls works just fine from any places. It's just that I cannot choose the SIM, unless I initiate from the Call History tab.
> 
> As for the workaround you mentioned, I don't understand what you mean by "You need to slide the notification menu and press the call you are trying to make from the contacts (careful not to hit the off button)".

Your describing exactly what happens to me. That was what I was trying to convey anyway. 

So what happens is that when you initiate a call from the contacts you get the name of your contact in a frozen screen. If you look up next to the time you have a phone icon. If you slide with your finger from there you'll see you call in the notifications tab. If you click on it you'll be able to choose the SIM. At least I can.

**Comment by inson1** [comment]
@muxina-credo @avatar1024 Hey SMT apps are dead. They were sold to ad company. F-Droid versions are still safe. Fork is here https://github.com/FossifyOrg

**Comment by muxina-credo** [comment]
@inson1 Are the FossifyOrg version the one that is fed into F-droid?

**Comment by inson1** [comment]
@muxina-credo No. Its the fork. Fork is if someone forks the poject. (copies the project and starts developing). First release should be in 2-3 weeks. :)

**Comment by muxina-credo** [comment]
@inson1 My question was that this forked version probably will come under a different name right? Just wanted to know how I'll be able to identify the new version. Thank you for letting me know that this project sold out to an ad company. It's always a shame when this happens.

**Comment by inson1** [comment]
@muxina-credo There is voting right now. Maybe there will be more pools. Right now winner is Pure - Pure Dialer, Pure Contacts .........
